Formula/psfex.rb

```diff
@@ -0,0 +1,26 @@
+class Psfex < Formula
+  homepage "https://www.astromatic.net/software/psfex"
+  url "https://www.astromatic.net/download/psfex/psfex-3.17.1.tar.gz"
+  sha256 "53f1b449ab7da7e6e0a989c41b82885f52c8f08270ceb4378bb1ec7ef754af89"
+
+  depends_on "fftw"
+  depends_on "plplot"
+  depends_on "autoconf" => :build
+
+  # this patches make the changes needed to compile with the Accelerate
+  # framework for linear algebra routines. Based on sextractor.rb recipe.
+  patch do
```

Comment:
Has this patch been submitted upstream ?

Reply:
If the software builds without patch using `openblas`, we might opt for that. Accelerate has issues.